build/dsc/templates/dsc_type.rb.erb

```diff
@@ -4,6 +4,7 @@ Puppet::Type.newtype(:dsc_<%= resource.friendlyname.downcase %>) do
   require Pathname.new(__FILE__).dirname + '../../' + 'puppet/type/base_dsc'
 
   provide :powershell, :parent => Puppet::Type.type(:base_dsc).provider(:powershell) do
+    confine :powershell_major_version => '5'
```

Comment:
> I think it's also nice to be inline with our documentation, which states lcm_config is optional. On earlier WMF5 releases, lcm_config is mandatory.

@jpogran and I talked about whether to add the line about it being mandatory still for Windows 10. He talked me out of adding it, but it's worth noting that it is still mandatory for Windows 10 users until RTM comes out.